models/ldndc/R/model2netcdf.LDNDC.R

```diff
@@ -0,0 +1,193 @@
+    
+    print(paste("---- Prosessing year: ", y)) # debugging
+    
+    # Subset data for prosessing
+    sub.ldndnc.out <- subset(ldndc.out, Year == y)
+    dayfrac <- 1/out_day
+    step <- head(seq(0,1, by = dayfrac), -1)
+    
+    
+    
+    # Generate start/end dates for processing
+    if (y == strftime(start_date, "%Y")) {
+      begin_date <- lubridate::yday(start_date)
+    } else {
+      begin_date <- 1
+    }
+    
+    if (y == strftime(end_date, "%Y")) {
+      end_d <- lubridate::yday(end_date)
+    } else {
+      end_d <- PEcAn.utils::days_in_year(y)
+    }
+    
+    ## Subset the years we are interested in
+    sub.ldndnc.out <- subset(sub.ldndnc.out, Day >= begin_date & Day <= end_d)
+    
+    
+    # Create the tvals that are used in nc-files
+    tvals <- sub.ldndnc.out[["Day"]] + sub.ldndnc.out[["Step"]] /out_day -1
+    
+    
+    
+    
+    ## Outputs need to be an appropriate units, this can be done here
+    output <- list()
+    # NEE value is on kg, so change it mg (*1 000 000), then change ha to m2 (/10 000) and then day to seconds (86400)
+    #output[[1]] <- sub.ldndnc.out$dC_NEE.kgCha.1. * 100 / timestep.s
+    
+    # Kilogram of total soil carbon in a m2
+    #output[[2]] <- sub.ldndnc.out$C_total.kgCha.1. / 10000
+    
+    # LAI
+    output[[1]] <- sub.ldndnc.out$lai
+    
+    # Photosynthesis rate - GPP
+    output[[2]] <- sub.ldndnc.out$dC_co2_upt.kgCm.2.
+    
+    #### Declare netCDF variables ####
+    t <- ncdf4::ncdim_def(name = "time",
+                          longname = "time",
+                          units = paste0("days since ", y, "-01-01 00:00:00"), #00:00:00
+                          vals = tvals,
+                          calendar = "standard",
+                          unlim = TRUE)
+    
+    
+    
+    lat <- ncdf4::ncdim_def("lat", "degrees_north", vals = as.numeric(sitelat), 
+                            longname = "station_latitude")
+    lon <- ncdf4::ncdim_def("lon", "degrees_east", vals = as.numeric(sitelon), 
+                            longname = "station_longitude")
+    dims <- list(lon = lon, lat = lat, time = t)
+    time_interval <- ncdf4::ncdim_def(name = "hist_interval", 
+                                      longname="history time interval endpoint dimensions",
+                                      vals = 1:2, units="")
+    
+    
+    
+    ## Declare netCDF variables ##
+    nc_var <- list()
+    #nc_var[[1]] <- PEcAn.utils::to_ncvar("NEE", dims)
+    #nc_var[[2]] <- PEcAn.utils::to_ncvar("TotSoilCarb", dims)
+    nc_var[[1]] <- PEcAn.utils::to_ncvar("LAI", dims)
+    nc_var[[2]] <- PEcAn.utils::to_ncvar("GPP", dims)
+    
+    
+    
+    ## Output netCDF data
+    nc <- ncdf4::nc_create(file.path(outdir, paste(y, "nc", sep = ".")), nc_var)
+    ncdf4::ncatt_put(nc, "time", "bounds", "time_bounds", prec = NA)
+    
+    
+    
+    for(i in seq_along(nc_var)){
+      ncdf4::ncvar_put(nc, nc_var[[i]], output[[i]])
```

Comment:
please also add the `varfile`, e.g. see [here](https://github.com/PecanProject/pecan/blob/develop/models/basgra/R/run_BASGRA.R#L486). I believe this file is used by some downstream plotting functions